src/generator-context.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.IO;
+
+#nullable enable
+
+namespace Xamarin.Generator.Traceability
```

Comment:
Are we still using Xamarin namespace or moving to Microsoft.MaciOS like in Nnyeah?

Reply:
Generator is in the Xamarin name space, and a lot of stuff is still there. nnyeah is new this code has other constrains, changing the namespace is not a thing I'd invest time on atm.